server/app/services/applicant/ApplicantData.java

```diff
@@ -114,4 +118,22 @@ public void setUserName(
       putString(WellKnownPaths.APPLICANT_LAST_NAME, lastName);
     }
   }
+
+  @Override
+  public String asJsonString() {
+    if (!getFailedUpdates().isEmpty()) {
+      throw new IllegalStateException(
+          "data cannot be serialized since there " + "were failed updates");
```

Comment:
nit: 
```suggestion
          "data cannot be serialized since there were failed updates");
```

Reply:
Strange. I imagine this was some form of me doing line splitting and bin/fmt deciding I shouldn't have done it that way.